wfn/uri.go

```diff
@@ -125,6 +133,12 @@ func pack(ss []string) string {
 // - unquoted special characters are mapped to their special forms.
 func bindValueURI(s string) string {
 	var out []byte
+	switch s {
```

Comment:
@skogtwin it passed because `bindValueURI("-")` was `""`

Reply:
And also, why? These constants are already that, NA="-", Any="",
```
switch s {
case NA, Any:
  return s
}
```
But again, I don't see why this is necessary? Just add it to the first if case on line 144, or create a new one `} else if s == NA || s == Any { out = append(out, b) }

Reply:
This is coincidental, so I'd rather keep it explicit.